monkey/monkey_island/cc/setup/mongo/mongo_db_process.py

```diff
@@ -47,3 +50,11 @@ def stop(self):
                 f"MongoDB did not terminate gracefully and will be forcefully killed: {te}"
             )
             self._process.kill()
+
+    def is_running(self):
+        if self._process.poll() is not None:
+            return True
```

Comment:
1. If `poll()` returns None, that means the process _is_ running, so the logic here is backwards.
1. Prefer positive condition over negative (e.g. prefer `is None` over `is not None`)
2. Make all return values explicit. Something named `is_running()` should always return a boolean. Currently, if the process is not running, the function implicitly returns `None`. While this will result in effectively the same as `False` for most of our use cases, it could cause issues down the road.

```suggestion
    def is_running(self):
        if self._process.poll() is None:
            return True
            
        return False
```